### DISCUSSION TITLE
Trying to use GPIO but have a error

### DISCUSSION THREAD

**Discussion by 10thCrusade** [xpack-dev-tools/qemu-arm-xpack#27] [Q&A] [answered]
I'm beginner in use STM32CubdeIDE, after trying to attach a button to use with GPIO have a error like :
stm32:gpio-peripheral: Write of size 4 at offset 0x10 on disabled peripheral, ignored. 
How to fix it? Dont found similar problem or smth.
![image](https://github.com/xpack-dev-tools/qemu-arm-xpack/assets/126684740/cfc68bb4-153a-44a5-b0b0-52d7f7da852c)
![image](https://github.com/xpack-dev-tools/qemu-arm-xpack/assets/126684740/78050e81-6258-49a5-a579-ff55f33d09af)

**Comment by ilg-ul** [comment, marked as answer]
> Write of size 4 at offset 0x10 on disabled peripheral

How about enabling the peripheral?
